app/server/routes/v1/user.js

```diff
@@ -140,6 +141,17 @@ router.patch("/", async (req, res) => {
       "communities",
     ]);
 
+    const firstName = req.body.firstName || documents.user.firstName;
+    const lastName = req.body.lastName || documents.user.lastName;
+
+    if (firstName && lastName) {
```

Comment:
what happens if the user is Cher or Ronaldo and only has a first name?

Reply:
then it uses their username

Reply:
Nevermind I see what you mean. If the person who created the account only has a first name. I'll make that change to allow only setting a first name.